runtime/gc_vlhgc/GlobalMarkCardScrubber.hpp

```diff
@@ -190,7 +190,7 @@ class MM_ParallelScrubCardTableTask : public MM_ParallelTask
 private:
 	MM_CycleState * const _cycleState; /**< Current cycle state information */
 	bool _timeLimitWasHit;	/**< true if any requests to check the time came in after we had hit our time threshold */
-	const I_64 _timeThreshold;	/**< The millisecond which represents the end of this increment */
+	const U_64 _timeThreshold;	/**< The millisecond which represents the end of this increment */
```

Comment:
Is this variable stored ticks instead of milliseconds now? If so, comment should be changed. Here and below

Reply:
Fixed, thanks!

Reply:
Does not seem to be fixed yet. 

Note, there a few if them to fix. For example, comment above declaration markScan.

Reply:
Interesting, it seems like my changes are not reflected here.

Reply:
I'm not sure why they are not there last Friday, but they should be fine now.